ros/processor/garbage_collector.py

```diff
@@ -0,0 +1,33 @@
+from ros.lib.app import app, db
+from ros.lib.models import PerformanceProfile
+from datetime import datetime, timedelta
+from ros.lib.config import GARBAGE_COLLECTION_INTERVAL
+import time
+import logging
+
+logging.basicConfig(
```

Comment:
It seems rest of the processors also using the same configs. 
Will it be possible to have a separate helper function which does logger initialization?